ecosystem/indexer-grpc/indexer-transaction-generator/README.md

```diff
@@ -9,6 +9,7 @@ This tool is to generate transactions for testing purpose.
 ### Config
 
 ```YAML
+# Config to import transactions onchain.
```

Comment:
I wonder how the config file would look like in prod, or when we have this Ci/CD with all transactions included.

Reply:
maybe under `indexer-test-transactions`? and one file looks like this, wdyt?

Reply:
updated; we have both README and config_template to showcase the usage.